components/pricing_modal/content.tsx

```diff
@@ -117,12 +119,16 @@ function Card(props: CardProps) {
                     {props.planExtraInformation && props.planExtraInformation}
                 </div>
                 <div>
-                    <button
-                        id={props.id + '_action'}
-                        className={`plan_action_btn ${props.buttonDetails.disabled ? ButtonCustomiserClasses.grayed : props.buttonDetails.customClass}`}
-                        disabled={props.buttonDetails.disabled}
-                        onClick={props.buttonDetails.action}
-                    >{props.buttonDetails.text}</button>
+                    {props.customButtonDetails ? props.customButtonDetails : (
```

Comment:
Maybe this should fallback to the buttonDetails (which would have cta to contact sales)?